code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -82,6 +85,8 @@
 
 /obj/item/weapon/storage/box/survival/engineer/New()
 	..()
+	icon_state = "box_eva"
```

Comment:
Boy I knew when I did this quickly, easily, and without extra work it'd end up being people bitching about me doing what whoever coded it did.